src/utils/emissions/PollutantsInterface.h

```diff
@@ -129,7 +129,19 @@ class PollutantsInterface {
                 return myEmissionClassStrings.get(eClass);
             }
             std::string eclower = eClass;
-            std::transform(eclower.begin(), eclower.end(), eclower.begin(), tolower);
+	    /* 
+	       For some compilers, std::tolower cannot be resolved correctly, resulting in error messages
+	       like "No matching function found ... unresolved overloaded function type.", see e.g.
+               https://stackoverflow.com/questions/5539249. The problem may be fixed by spacifying ::tolower,
+	       the global namespace version of the function that has no overloads.
+
+	       Similarly, https://en.cppreference.com/w/cpp/string/byte/tolower siggests that one should not
+	       use any of the functions defined in <cctype> with standard slgorithms (like `transform`) when
+	       the iterator type is `char` or `signed char` -- we shall convert the value to `unsigned char`
+	       first:
+	        std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c){ return std::tolower(c); });
+	    */
+            std::transform(eclower.begin(), eclower.end(), eclower.begin(), ::tolower);
```

Comment:
My Ubuntu chokes on the last parameter being `tolower`, but I suppose that master branch compiles without problems on Linux. Could be namespace related.